microsoft_atp/help.md

```diff
@@ -19,12 +19,6 @@ This plugin utilizes the [Microsoft ATP API](https://docs.microsoft.com/en-us/wi
 
 ## Setup
 
-This plugin uses the Windows Defender ATP API. It will use an Azure application to connect to the API and run
```

Comment:
Add this paragraph back - the tooling removed it - it's a bug so will have to revert or commit it back again.